packages/react/src/components/DatepickerInput/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+const DatepickerInput = React.forwardRef(
+  ({ onClick, placeholder, value, isInvalid }, ref) => (
+    <div
+      role='button'
+      tabIndex={0}
```

Comment:
It should be a `button` instead, better semantics than a div with role button. You may need to add two more lines of CSS to make it match but I'm doing those changes where I can for V2

Reply:
Fixed :+1: